tests/Plaid.Demo/Startup.cs

```diff
@@ -2,28 +2,27 @@
 using Going.Plaid.Demo.Middleware;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 
 namespace Going.Plaid.Demo
 {
 	public class Startup
 	{
+		private readonly IConfiguration _config;
+		public Startup(IConfiguration configuration)
+		{
+			_config = configuration;
+		}
+
 		public void ConfigureServices(IServiceCollection services)
 		{
 			services.AddMvc(o => o.EnableEndpointRouting = false);
 			services.AddHttpClient();
-			services.AddSingleton<PlaidCredentials>();
-			services.AddSingleton(
-				sp =>
-				{
-					var credentials = sp.GetRequiredService<PlaidCredentials>();
-					return new PlaidClient(
-						credentials.Environment,
-						credentials.ClientId,
-						credentials.Secret,
-						httpClientFactory: sp.GetRequiredService<IHttpClientFactory>());
-				});
+			services.Configure<PlaidCredentials>(_config.GetSection(PlaidOptions.SectionKey));
```

Comment:
There should be a way to inform the service collection to use the `PlaidCredentials` instance for `PlaidOptions` as well instead of creating two configurations.

Reply:
As far as I know, you need to run a Configure<TClass> for each IOptions<TClass> you want to use in dependency injection. If we added the classes directly to the ServiceProvider as Singletons then referenced them in the constructors without IOptions wrappers, then we could use the same instance of PlaidCredentials where the DI goes looking for any PlaidOptions instance ... but I don't know if that's possible with IOptions.

Reply:
That's fine. I probably wouldn't recommend `PlaidCredential` inheriting from `PlaidOptions` in general, but it makes sense for how we're setting up the demo.